src/components/Review.jsx

```diff
@@ -61,76 +65,121 @@ class Review extends Component {
               <Row>
                 <Col lg={2}>
                   <h2>
-                    <Badge
-                      variant="secondary"
-                      style={{
-                        backgroundColor: badgeColor(
-                          this.state.review.rating.overall
-                        )
-                      }}
+                    <OverlayTrigger
+                      key={"top"}
+                      placement={"top"}
+                      overlay={
+                        <Tooltip id={`tooltip-top`}>
+                          {labels[4][this.state.review.rating.overall - 1]}
+                        </Tooltip>
+                      }
                     >
-                      {this.state.review.rating.overall}
-                    </Badge>
+                      <Button
+                        variant={badgeVariant(this.state.review.rating.overall)}
+                      >
+                        {this.state.review.rating.overall}
+                      </Button>
+                    </OverlayTrigger>
                   </h2>
                 </Col>
                 <Col>
                   <Row>
                     <Col>
                       <span>
-                        <Badge
-                          variant="secondary"
-                          style={{
-                            backgroundColor: badgeColor(
-                              this.state.review.rating.difficulty
-                            )
-                          }}
+                        <OverlayTrigger
+                          key={"top"}
+                          placement={"top"}
+                          overlay={
+                            <Tooltip id={`tooltip-top`}>
+                              {
+                                labels[0][
+                                  this.state.review.rating.difficulty - 1
+                                ]
+                              }
+                            </Tooltip>
+                          }
                         >
-                          {this.state.review.rating.difficulty}
-                        </Badge>
+                          <Button
+                            variant={badgeVariant(
+                              this.state.review.rating.difficulty
+                            )}
+                            size="sm"
+                          >
+                            {this.state.review.rating.difficulty}
+                          </Button>
+                        </OverlayTrigger>
                         &nbsp;Difficulty
```

Comment:
These words (Difficulty, Attendance, etc) shouldn't be hardcoded here. Take a look at the `desc` array inside `Composer.jsx`. It would be better to handle this in the same way that `labels` are handled, i.e. put the desc array in the `labels.js` file, and then import it into both components.